"Get Involved" done in landing

Zarko36 opened the pull request:
<img width="534" alt="image" src="https://github.com/acm-ucr/art-factory-website/assets/74474117/6d414878-0f97-415d-a04e-ec8ebe226964">

* Worked on "Get Involved" section in landing
* Completed using react icons and coloring the icons and text using colors found in tailwind.config.js
* Imported react icons directly into Landing.jsx
* Took note that the link icon is smaller compared to the other icons on figma

Zarko36 committed:
"Get Involved" done in landing

stormyy00 commented on src/app/page.js:
```diff
@@ -1,11 +1,13 @@
     <div className="w-full flex flex-col items-center">
       <p>Home</p>
       <Button text="LEARN MORE" link="/about" />
+      <Landing />
```
move it to Link.jsx and remove Links.jsx as well

stormyy00 commented on src/components/Landing.jsx:
```diff
@@ -1,9 +1,39 @@
+      <hr className="border-2 border-art-pink-200 w-12" />
+      <div className="flex py-5 space-x-10">
+        <div className="flex flex-col items-center">
+          <FaDiscord size="48px" color="#E893C5" />
```
too many divs move turn this into a  .map() to make it more cleaner since you essentially reusing (flex flex-col ) and create a socials.js file in the data folder

stormyy00 replied:
react icons can be classified as text so use classname"text-<size> text-<color>" 
also try to find the closest tw.config color that matches to the hex code

Zarko36 committed:
Optimizations

Zarko36 commented:
<img width="555" alt="image" src="https://github.com/acm-ucr/art-factory-website/assets/74474117/38a91cde-7356-4b67-aa5b-25d1e8df317a">

* Made the edges of the header round
* Moved Landing component to Link.jsx
* Deleted Links.jsx
* Condensed divs into a .map
* Used pyNumb to make sure smaller link icon align with the other regular sized icons
* Used icons as text to apply text-art

stormyy00 committed:
adjusted socials component
